### PR TITLE
Adds bartending shotgun rounds to the game (April fools)

### PR DESCRIPTION
**Pull request by Blackern5000**
## About the PR
ever since I nerfed the bartender's beanbags I've thought "man, this still sucks, bartenders shouldn't get stamina damage at all!"
I have replaced the beanbag rounds in the bartender's storage with bartending rounds which deal exactly 0 stamina damage. Balance at last.

## Why / Balance
Stamina damage is stupid >:(

## Media
https://github.com/space-wizards/space-station-14/assets/140123969/e78f553f-5677-402b-934d-e57d43b05b7a
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Bartender's shotguns no longer deal any stamina damage, no longer must we worry about being stunned in the bar.

### CODE REVIEW

**Comment by ImWeax** [comment]
Seems balanced to me

**Comment by IamVelcroboy** [comment]
Bartender's RPG when?

**Comment by deltanedas** [comment]
i thought this was some bullshit like shooting someone then they have a drink glass appear in their hand but no

**Comment by Krunklehorn** [comment]
> i thought this was some bullshit like shooting someone then they have a drink glass appear in their hand but no

Like shot~~glass~~gun rounds? Genius.

File: Resources/Prototypes/Entities/Objects/Fun/immovable_rod.yml [hunk 1]
```diff
@@ -60,3 +60,14 @@
   components:
   - type: ImmovableRod
     randomizeVelocity: false
+
+- type: entity
+  parent: ImmovableRodKeepTilesStill
+  id: ImmovableRodBartender
+  suffix: Bartender
+  name: immovable rod
```
Comment (FairlySadPanda) [inline Resources/Prototypes/Entities/Objects/Fun/immovable_rod.yml:68]: "immovable bar" rather than "rod"

**Comment by Terraspark4941** [comment]
that was so fucking unexpected, rofl
this year's april fools is going to be FUN

**Comment by metalgearsloth** [comment]
There's april fool's and there's make the game unplayable, think of something funny.

**Comment by ImWeax** [comment]
I for one found this quite funny

**Comment by CatMagic1** [comment]
Agreed. Maintainers dunno what fun is.

**Comment by ThatOneGoblin25** [comment]
number changes aren't funny plus reskinned practice round, you should have bartending rounds work like tranq rounds but add alcohol into people, you'know, something that has expressed impact rather than taking away stuff.

**Comment by dolgovmi** [comment]
> number changes aren't funny plus reskinned practice round, you should have bartending rounds work like tranq rounds but add alcohol into people, you'know, something that has expressed impact rather than taking away stuff.

Did you view the video that comes along with the PR

**Comment by K-Dynamic** [comment]
Meanwhile from 6 months ago (not even april fools): old gods shotgun shells :trollface: 

https://youtu.be/JmAyUnocbqY?feature=shared

**Pull request closed without merging**